refbox/translations/es/refbox.ftl

```diff
@@ -0,0 +1,246 @@
+stop-clock-last-2-min = Detener reloj en los últimos 2 minutos: Desconocido
+refs = Árbitro principal: Desconocido
+        Timer: Desconocido
+        Árbitro de agua 1: Desconocido
+        Árbitro de agua 2: Desconocido
+        Árbitro de agua 3: Desconocido
+
+
+# List Selecters
+select-tournament = SELECCIONAR TORNEO
+select-court = SELECCIONAR CANCHA
+select-game = SELECCIONAR PARTIDO
+
+# Main View
+add-warning = AÑADIR AVISO
+add-foul = AÑADIR FALTA
+start-now = EMPEZAR AHORA
+end-timeout = TERMINAR TIEMPO DE ESPERA
+warnings = AVISOS
+penalties = PENALIZACIONES
+score = PUNTUACIÓN
+
+# Penalties
+penalty_color = { $color } PENALIZACIONES
+
+# Score edit
+final-score = Por favor ingrese la puntuación final
+
+# Shared Elements
+## Timeout ribbon
+end-timeout-line-1 = TERMINAR
+end-timeout-line-2 = TIEMPO MUERTO
+switch-to = CAMBIAR A
+ref = ÁRBITRO
+penalty = PENALIZACIÓN
+shot = TIRO
+pen-shot = TIRO PENAL
+## Penalty string
+served = Servido
+dismissed = DESCARTADO
+## Config String
+error = Error ({ $number })
+none = Ninguno
+two-games = Último partido: { $prev_game }, Próximo partido: { $next_game }
+one-game = Juego: { $game }
+teams = { -dark-team-name } Equipo: { $dark_team }
+    { -light-team-name } Equipo: { $light_team }
+game-config = Duración de la mitad: { $half_len },  Duración del medio tiempo: { $half_time_len }
+    Muerte súbita permitida: { $sd_allowed },  Tiempo Extra Permitido: { $ot_allowed }
+    Tiempos muertos de equipo permitidos por mitad: { $team_timeouts }
+    Detener reloj en los Últimos 2 minutos: { $stop_clock_last_2 }
+ref-list = Árbitro principal: { $chief_ref }
+    Timer: { $timer }
+    Árbitro de agua 1: { $water_ref_1 }
+    Árbitro de agua 2: { $water_ref_2 }
+    Árbitro de agua 3: { $water_ref_3 }
+## Game time button
+next-game = PRÓXIMO JUEGO
+first-half = PRIMERA MITAD
+half-time = MEDIO TIEMPO
+second-half = SEGUNDA MITAD
+pre-ot-break-full = DESCANSO PREVIO AL TIEMPO EXTRA
+overtime-first-half = PRIMERA MITAD DEL TIEMPO EXTRA
+overtime-half-time = MEDIO TIEMPO DEL TIEMPO EXTRA
+overtime-second-half = SEGUNDA MITAD DEL TIEMPO EXTRA
+pre-sudden-death-break = DESCANSO PREVIO A MUERTE SÚBITA
+sudden-death = MUERTE SÚBITA
+ot-first-half = PRIMERA MITAD DEL TIEMPO EXTRA
+ot-half-time = MEDIO TIEMPO DEL TIEMPO EXTRA
+ot-2nd-half = SEGUNDA MITAD DEL TIEMPO EXTRA
+white-timeout-short = T/M BLANCO
+white-timeout-full = TIEMPO MUERTO BLANCO
+black-timeout-short = T/M NEGRO
+black-timeout-full = TIEMPO MUERTO NEGRO
+ref-timeout-short = T/M ÁRBITRO
+penalty-shot-short = TIRO PENAL
+## Make penalty dropdown
+infraction = INFRACCIÓN
+## Make warning container
+team-warning-abreviation = A
+
+# Time edit
+game-time = TIEMPO DE JUEGO
+timeout = TIEMPO MUERTO
+Note-Game-time-is-paused = Nota: El tiempo de juego está pausado mientras está en esta pantalla
+
+# Warning Fouls Summary
+fouls = FALTAS
+edit-warnings = EDITAR AVISO
+edit-fouls = EDITAR FALTAS
+
+# Warnings
+color-warnings = { $color } AVISOS
```

Comment:
```suggestion
black-warnings = ADVERTENCIAS NEGRAS
white-warnings = ADVERTENCIAS BLANCAS
```
This was changed since you got the original english file